keyboards/kiwikey/kawii9v2/lib/oled.c

```diff
@@ -0,0 +1,251 @@
+// Copyright 2023 KiwiKey
+// SPDX-License-Identifier: GPL-2.0-or-later
+ 
+#include QMK_KEYBOARD_H
```

Comment:
The necessary files (if any) should be directly included here, not the `QMK_KEYBOARD_H` macro.
```suggestion
```